package/src/contexts/attachmentPickerContext/AttachmentPickerContext.tsx

```diff
@@ -114,8 +113,21 @@ export const AttachmentPickerProvider = ({
   );
 };
 
-export const useAttachmentPickerContext = () =>
-  useContext(AttachmentPickerContext) as unknown as AttachmentPickerContextValue;
+export const useAttachmentPickerContext = () => {
+  const contextValue = useContext(
+    AttachmentPickerContext,
+  ) as unknown as AttachmentPickerContextValue;
+
+  if (!contextValue) {
+    console.error(
+      `The useAttachmentPickerContext hook was called outside the AttachmentPickerContext provider. Make sure you have configured OverlayProvider component correctly - https://getstream.io/chat/docs/sdk/reactnative/basics/hello_stream_chat/#overlay-provider`,
+    );
+
+    return {} as AttachmentPickerContextValue;
+  }
+
+  return contextValue as AttachmentPickerContextValue;
```

Comment:
```suggestion
  return contextValue;
```

The extra assertion here seems redundant, and the type check seems to be fine without it when I test it out - do you get an error without it?

Reply:
Not sure of this as I didn't introduce this change in my PR and it was present from before. I will have to check.